build/00-embassy

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+printf "\n"
+printf "Welcome to\n"
+cat << "ASCII"
+,---.     |                        ,---.,---.
+|--- ,-.-.|---.,---.,---.,---.,   .|   |`---.
+|    | | ||   |,---|`---.`---.|   ||   |    |
+`---'` ' '`---'`---^`---'`---'`---|`---'`---'
+                              `---'
+ASCII
+printf "   %s (%s %s)\n" "$(uname -o)" "$(uname -r)" "$(uname -m)"
```

Comment:
add to this `EmbassyOS v$(embassy-cli version)) - $(embassy-cli git-info)`

Reply:
You mean something like this?
```
ASCII
printf "   %s (%s %s)\n" "$(uname -o)" "$(uname -r)" "$(uname -m)"
printf "   $(embassy-cli --version) - $(embassy-cli git-info)\n"
```
will result into:

![Screenshot from 2022-01-24 22-40-16](https://user-images.githubusercontent.com/3606313/150869564-0fe2a1a0-e1db-4612-90fb-ca5385fcecc7.png)

Reply:
Yes.

Reply:
There is currently a bug in embassy-cli that causes version not to work but that's exactly what I mean.